src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -278,12 +281,19 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   resolved(captchaResponse: string) {
-    const newResponse = captchaResponse
-      ? `${captchaResponse.substr(0, 7)}...${captchaResponse.substr(-7)}`
-      : captchaResponse;
-    this.captchaResponse += `${JSON.stringify(newResponse)}\n`;
-    if (this.captchaResponse) {
-      this.onSubmitSignUpForm();
+    if (captchaResponse) {
+      const httpOptions: HttpOptions = {
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      };
+      this.http.post('/validate/recaptcha?captchaResponse=' + captchaResponse, httpOptions)
+        .subscribe((data: any) => {
+          this.onSubmitSignUpForm();
+        }, (error) => {
+          this.toasterService.error(this.resourceService.messages.emsg.m0005);
```

Comment:
Log Telemetry Interact & error event for this. So that we can track these in Superset/Drup(to identify who are these users)